src/ngamsPlugIns/ngamsPlugIns/ngamsSdmMultipart.py

```diff
@@ -37,18 +37,22 @@
 implemented and NG/AMS configured to use it.
 """
 
+import cgi
+from datetime import date
 import logging
 import os
+import re
+import rfc822
```

Comment:
Beware that `rfc822` is deprecated since ptyhon 2.3, and has been removed from 3 in favour of the `email` module, so this will fail to run in python 3.